### ISSUE TITLE
preloadRefEd example window is blank

### ISSUE THREAD

**Issue by elrond79** [LumaPictures/pymel#271]
_From [Count....@gmail.com](https://code.google.com/u/113597931548292875171/) on March 11, 2011 12:51:18_

When the sample code run from the docs for pymel.core.system.preloadRefEd are run:

import pymel.core as pm

window = pm.window()
pm.paneLayout()
\# Result: ui.PaneLayout('window1|paneLayout2') #
pm.preloadRefEd()
\# Result: u'preloadRefEd1' #
pm.showWindow(window)

The window comes up blank, even with references in a loaded scene Describe the problem. MayaVersion: 2011 x64, Win7
PyMel 1.0.2

_Original issue: http://code.google.com/p/pymel/issues/detail?id=237_

**Comment by elrond79** [comment]
_From [tradigi...@gmail.com](https://code.google.com/u/116919639809502748349/) on March 11, 2011 13:43:11_

I'm getting the same problem on Win 32, Maya 2009, pymel 1.0.2

**Comment by elrond79** [comment]
_From [elron...@gmail.com](https://code.google.com/u/112796493857278763022/) on March 31, 2011 07:54:27_

I've confirmed this as a problem.
The issue seems to be with the root maya.cmds function - running the example code for maya.cmds.preloadRefEd:

import maya.cmds as cmds
window = cmds.window()
cmds.paneLayout()
cmds.preloadRefEd()
cmds.showWindow(window)

...gives the same result, and I haven't found any way to make it show up yet.

I submitted a bug report to autodesk, `#383339`, but after asking around, it seems it's a known issue that's been around since 2009.

Given that there's not much we can do to fix this if the underlying command isn't working at all (and that, presumably, once the maya.cmds version starts working, the pymel one should as well), I'm going to close this ticket.

**Status:** WontFix
